### PR TITLE
New package: Terminal-0.9.9

### PR DESCRIPTION
**Pull request by phossil**
Terminal emulator for GNUstep

### CODE REVIEW

**Comment by phossil** [comment]
```shell
phossil in void-packages on  Terminal.app took 36s ❯ ./xbps-src clean
=> xbps-src: removing autodeps, please wait...
=> xbps-src: cleaning up masterdir...
phossil in void-packages on  Terminal.app ❯ XBPS_CHECK_PKGS=yes ./xbps-src -a x86_64-musl pkg Terminal.app
=> xbps-src: updating repositories for host (x86_64)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating repositories for target (x86_64-musl)...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/x86_64-musl-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/nonfree/x86_64-musl-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/musl/debug/x86_64-musl-repodata' ...
=> xbps-src: updating software in / masterdir...
=> xbps-src: cleaning up / masterdir...
=> Terminal.app-0.9.9_1: removing autodeps, please wait...
=> Terminal.app-0.9.9_1: removing autocrossdeps, please wait...
=> Terminal.app-0.9.9_1: building [gnu-makefile] for x86_64-musl...
   [host] gnustep-make-2.8.0_1: found (https://alpha.de.repo.voidlinux.org/current)
   [host] gcc-objc-9.3.0_8: found (https://alpha.de.repo.voidlinux.org/current)
   [host] tar-1.32_3: found (https://alpha.de.repo.voidlinux.org/current)
   [host] which-2.21_3: found (https://alpha.de.repo.voidlinux.org/current)
   [target] gnustep-base-devel-1.27.0_2: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [target] gnustep-gui-devel-0.28.0_3: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [runtime] gnustep-base-1.27.0_2: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [runtime] gnustep-gui-0.28.0_3: found (https://alpha.de.repo.voidlinux.org/current/musl)
   [runtime] gnustep-back-0.28.0_1: found (https://alpha.de.repo.voidlinux.org/current/musl)
=> Terminal.app-0.9.9_1: installing host dependencies: gnustep-make-2.8.0_1 gcc-objc-9.3.0_8 tar-1.32_3 which-2.21_3 ...
=> Terminal.app-0.9.9_1: installing target dependencies: gnustep-base-devel-1.27.0_2 gnustep-gui-devel-0.28.0_3 ...
=> Terminal.app-0.9.9_1: removing autodeps, please wait...
=> Terminal.app-0.9.9_1: removing autocrossdeps, please wait...
=> ERROR: Terminal.app-0.9.9_1: failed to install 'gnustep-base-devel-1.27.0_2' dependency! (error 8)
gnustep-base-1.27.0_2: broken, unresolvable shlib `libobjc.so.4'
gnustep-gui-0.28.0_3: broken, unresolvable shlib `libobjc.so.4'
Transaction aborted due to unresolved shlibs.
=> ERROR: Please see above for the real error, exiting...
phossil in void-packages on  Terminal.app took 5s ✗ 
```

File: srcpkgs/Terminal.app/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'Terminal.app'
+pkgname=Terminal.app
```
Comment (ahesford) [inline srcpkgs/Terminal.app/template:2]: Package name should be `Terminal`

File: srcpkgs/Terminal.app/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+version=0.9.9
+revision=1
+wrksrc="Terminal-${version}"
```
Comment (ahesford) [inline srcpkgs/Terminal.app/template:5]: This can be removed with the proper package name.

File: srcpkgs/Terminal.app/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+hostmakedepends="gnustep-make gcc-objc"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui"
+short_desc="Terminal emulator for GNUstep"
+maintainer="Phosu Parsons <founderofjp@gmail.com>"
+license="GPL-2.0-only"
+homepage="http://gap.nongnu.org/terminal/"
+distfiles="${NONGNU_SITE}/gap/Terminal-${version}.tar.gz"
+checksum=27f81bbecdbd10cde8c080012cde4973bec20e551c58c29f353733cf03c220e9
+
```
Comment (ahesford) [inline srcpkgs/Terminal.app/template:17]: Remove the blank line here.

File: srcpkgs/Terminal.app/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'Terminal.app'
+pkgname=Terminal.app
+version=0.9.9
+revision=1
+wrksrc="Terminal-${version}"
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui"
```
Comment (ahesford) [inline srcpkgs/Terminal.app/template:10]: When I attempt to launch this, I get
```
020-12-17 13:33:08.904 Terminal[11578:11578] Did not find correct version of backend (libgnustep-back-028.bundle), falling back to std (libgnustep-back.bundle).
2020-12-17 13:33:08.904 Terminal[11578:11578] NSApplication.m:304  Assertion failed in initialize_gnustep_backend.  Unable to find backend back
Terminal: Uncaught exception NSInternalInconsistencyException, reason: NSApplication.m:304  Assertion failed in initialize_gnustep_backend.  Unable to find backend back
```
Perhaps you are missing some more `depends`?

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/Terminal.app/template (hunk 4 above)
Comment (abenson) [reply srcpkgs/Terminal.app/template:10]: `gnustep-back` most likely.

Comment (phossil) [reply srcpkgs/Terminal.app/template]: Seems functional in my Void VM, should be fixed now.

**Comment by abenson** [comment]
Update the commit message to match the package name. Looks good, otherwise.

**Pull request merged**